### DISCUSSION TITLE
Default behavior of a dragging window

### DISCUSSION THREAD

**Discussion by anametologin** [anametologin/krohnkite#31] [Polls]
1.  Can  get a float window by dragging (the old way) 
[5.webm](https://github.com/anametologin/krohnkite/assets/165245883/853fd308-bef4-4c4e-a4b2-9e05a393d74b)
2. Always be tiled, getting a float window only by shortcut. This way also very useful when you have multiple monitors with different resolution and prefer moving windows by mouse because KWin do not shrink the window automatically now-days.

**Comment by suhail-singh** [comment]
Are these poll options mutually exclusive? I.e., is the question whether to allow one to float windows only via shortcut (but not by dragging) or only via dragging (but not by shortcut)?

**Reply by anametologin** [reply]
No, this only applies dragging windows by mouse. Keyboard shortcuts are sacred.

**Reply by suhail-singh** [reply]
> Keyboard shortcuts are sacred

Thank you

**Comment by jjspierx** [comment]
For me, I use the keyboard shortcuts pretty much exclusively to move/order/resize my windows, and it works really well.  I have several windows types/classes to float automatically such as set sized windows like zoom dialogs, or kde settings, etc...However, every once in awhile in awhile I would like to float a window by dragging it. I would be in favor of this option.  I think the float option could be a smart float where depending on where the drop is, it could detect if it think you are just trying to move the window to another tile location, vs, dropping nowhere near and edge/corner where it is determines you mean to float the windows.
